exporter/datasetexporter/README.md

```diff
@@ -53,8 +64,17 @@ Make sure to provide the appropriate server host value in the `serverHost` attri
   - `retry_max_elapsed_time` (default = 300s): Is the maximum amount of time spent trying to send a buffer.
   - `retry_shutdown_timeout` (default = 30s): The maximum time for which it will try to send data to the DataSet during shutdown. This value should be shorter than container's grace period.
 - `logs`:
-    - `export_scope_info_on_event` (default = false): Include LogRecord scope information (if available) on the DataSet event.
+    - `export_resource_info_on_event` (default = false): Include LogRecord resource information (if available) on the DataSet event.
+    - `export_resource_prefix` (default = 'resource.attributes.'): If resource is included then the following string will be used as prefix.
+    - `export_scope_info_on_event` (default = true): Include LogRecord scope information (if available) on the DataSet event.
+    - `export_scope_prefix` (default = 'scope.attributes.'): If scope is included then the following string will be used as prefix.
+    - `export_separator` (default = '.'): When attribute contains some nested structures (maps, arrays) this string is used to separate them during flattening.
+    - `export_distinguishing_suffix` (default = '_'): If there is a collision between event attributes the following string is used as suffix for distinguishing them.
```

Comment:
A suffix string to resolve naming collisions when flattening with `export_separator`.

Reply:
Fixed